docs/photoniq/prerendering/monitor-prerendering/view-prerendering-activity.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 10
+title: View Prerendering Activity
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You have several options for viewing Dynamic Prerendering activity, either aggregated or per origin.
+
+- **Activity tab** - You can view activity metrics for all origins or for one origin on the Activity tab in the web console. You can view metrics for the last day, the last week, or the last month.
+  ![Prerendering Activity Tab](/img/prerendering/activity-tab.png)
+- **REST API** - Macrometa provides several API calls that allow you to view metrics for all origins or for one origin. You can also sort and set any time frame that you need.
+
+## View Prerendering Activity
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view prerendering activity in the web console.
+
+1. Log in to your Dynamic Prerendering dashboard.
+2. Click **Activity**.
+3. (Optional) Select a specific origin. Default is **All origins**.
+4. (Optional) Select a time frame. Default is **Last Day**.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+- [Get render-stats metrics](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-metrics-stats-render/get)
+- [Get render-stats metrics by origin](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-metrics-stats-render-origin/get)
+- [Get usage metrics](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-usage-aggregationType/get)
+- [Get usage metrics by origin](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-usage-aggregationType---origin/get)
+
+</TabItem>
+</Tabs>
+
+## Activity Metrics
+
+Dynamic Prerendering provides the following metrics. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
+
+- **Renders** - Total number of renders performed during the selected time.
+- **Requests per Second** - Number of renders performed per second during the selected time.
+- **Requests** - Total number of page requests received during the selected time.
+- **Cache Hits** - Number of requests served from the cache.
```

Comment:
In some fields we have `during the selected time`. In others, not.
I think we have two options here:
1. Add `during the selected time` to all fields.
2. Remove `during the selected time` from all fields and clarify in a separate paragraph that all fields are related to the `selected time`.